src/Number.php

```diff
@@ -157,4 +157,25 @@ public function getIntegerRoundingMultiplier()
 
         return '1';
     }
+
+    /**
+     * @param string|int $number
+     *
+     * @return bool
+     */
+    public static function isInteger($number)
+    {
+        if (filter_var($number, FILTER_VALIDATE_INT) !== false) {
+            return true;
+        }
+
+        $invalid = array_filter(
```

Comment:
Not sure this is the fastest, but we can improve that if necessary without breaking API. I will try to write some benchmarks.